collect_app/src/main/java/org/odk/collect/android/utilities/SnackbarUtils.java

```diff
@@ -20,7 +20,7 @@
 import android.widget.TextView;
 
 public final class SnackbarUtils {
-    private static final int LONG_DURATION_MS = 3500; // in milli seconds
+    private static final int LONG_DURATION_MS = 3500; // in milliseconds
```

Comment:
Sorry I didn’t notice this before. Since ms is an abbreviation for millisecond, and MS is part of the name, the comment is redundant, n'est-ce pas?

Reply:
Sure